pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -421,6 +421,11 @@ func (r *shootReconciler) runReconcileShootFlow(ctx context.Context, o *operatio
 			Fn:           flow.TaskFn(botanist.DeployIngressDNSResources).DoIf(!o.Shoot.HibernationEnabled),
 			Dependencies: flow.NewTaskIDs(nginxLBReady),
 		})
+		_ = g.Add(flow.Task{
+			Name:         "Cleaning up orphaned DNSRecord secrets",
+			Fn:           flow.TaskFn(botanist.CleanupOrphanedDNSRecordSecrets),
+			Dependencies: flow.NewTaskIDs(deployInternalDomainDNSRecord, deployExternalDomainDNSRecord, deployOwnerDomainDNSRecord, deployIngressDomainDNSRecord)})
+
```

Comment:
Let's align the formatting:
```suggestion
			Dependencies: flow.NewTaskIDs(deployInternalDomainDNSRecord, deployExternalDomainDNSRecord, deployOwnerDomainDNSRecord, deployIngressDomainDNSRecord)
		})
```